src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -11,7 +11,8 @@
     private String[] attributesToCrop;
     private int cropLength;
     private String[] attributesToHighlight;
-    private String filter;
+    private String[] filter;
```

Comment:
I'm sure you must be aware of this, but adding it just in case here: 
This change might affect users to update their client code, I explored other options, but this seemed like the best way

Reply:
I took some time because I think you are right, this is probably the best way to go, but it also causes a `breaking change` in the use of the SDK.